diff --git a/apps/mobile/node_modules/expo-image-picker b/apps/mobile/node_modules/expo-image-picker
new file mode 120000
index 0000000000000000000000000000000000000000..5be19f274723eb969d28d5d72a0f6cd203d3b4ac
--- /dev/null
+++ b/apps/mobile/node_modules/expo-image-picker
@@ -0,0 +1 @@
+../../../node_modules/.bun/expo-image-picker@55.0.12+0aa7a96d9fb94eb0/node_modules/expo-image-picker
\ No newline at end of file
diff --git a/.eslintrc.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/.eslintrc.js
deleted file mode 100644
index 2720197860feb8f9ea1c32274d44095ec956d508..0000000000000000000000000000000000000000
diff --git a/CHANGELOG.md b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/CHANGELOG.md
deleted file mode 100644
index 889ba681efa3d5cbcbb68f427156eba20d13899d..0000000000000000000000000000000000000000
diff --git a/README.md b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/README.md
deleted file mode 100644
index c9d98c98f33a8bf2edc24a7b5f147ac5ba8ea469..0000000000000000000000000000000000000000
diff --git a/android/build.gradle b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/build.gradle
deleted file mode 100644
index 99541666c38f3fd521d7412a6fc5957bfee49e4e..0000000000000000000000000000000000000000
diff --git a/android/src/main/AndroidManifest.xml b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/AndroidManifest.xml
deleted file mode 100644
index 58e2df1c145828aa9c15fc2fa856beeed5fe00ab..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ExpoCropImageActivity.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ExpoCropImageActivity.kt
deleted file mode 100644
index 59ca71889cd1813acb3be36ae18158eb1dd8827a..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ExpoCropImageUtils.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ExpoCropImageUtils.kt
deleted file mode 100644
index a876bbc1ae56f142f21e3dcddf9f18539128b376..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ImagePickerConstants.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ImagePickerConstants.kt
deleted file mode 100644
index cd6d77e5c9a1c29548207404e24dc18e147817ab..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ImagePickerExceptions.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ImagePickerExceptions.kt
deleted file mode 100644
index ae28b06d37e81919340ad9acef4cd1db95cdc96f..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt
deleted file mode 100644
index 50663d81d25054ebf5456fa465ba4d17a14bbe34..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ImagePickerOptions.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ImagePickerOptions.kt
deleted file mode 100644
index 51b67e4b3a84e0002644dbe1fe556436285f873e..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ImagePickerResponse.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ImagePickerResponse.kt
deleted file mode 100644
index 5768d92c340c0f480161f35fa312c6a34e27e3ba..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/ImagePickerUtils.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/ImagePickerUtils.kt
deleted file mode 100644
index cdb023c4a64aacc75c7d91bd4962af4e1556cb9b..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/MediaHandler.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/MediaHandler.kt
deleted file mode 100644
index 22076ad71d4409e7f6173bb4e4187b092d002ea5..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/contracts/CameraContract.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/contracts/CameraContract.kt
deleted file mode 100644
index 6d11883db8f1aac9cba076a275769d05bdfd3e59..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/contracts/ContractsUtils.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/contracts/ContractsUtils.kt
deleted file mode 100644
index 9d256d158c4a54b306478a55c962df87f1913482..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/contracts/CropImageContract.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/contracts/CropImageContract.kt
deleted file mode 100644
index 6337340258f2e7ed6bc86225ed7351b458a8a5e3..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/contracts/ImageLibraryContract.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/contracts/ImageLibraryContract.kt
deleted file mode 100644
index e0dd37cff95b1fdeaa45db167a2ead41ac27f759..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/exporters/CompressionImageExporter.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/exporters/CompressionImageExporter.kt
deleted file mode 100644
index 9ffeaaa59460aaf519a4bbe5f6eb8a42016220d0..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/exporters/DimensionsExporter.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/exporters/DimensionsExporter.kt
deleted file mode 100644
index 590cc6bfa613072836223fd284a671cb0c5477f0..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/exporters/ImageExporter.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/exporters/ImageExporter.kt
deleted file mode 100644
index bb1ba39672bb50b6b0e6f31922496cb5e4a18cbc..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/exporters/RawImageExporter.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/exporters/RawImageExporter.kt
deleted file mode 100644
index 92ae2f25c7c3fc044d66f5df80635bb4232aa82f..0000000000000000000000000000000000000000
diff --git a/android/src/main/java/expo/modules/imagepicker/fileprovider/ImagePickerFileProvider.kt b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/java/expo/modules/imagepicker/fileprovider/ImagePickerFileProvider.kt
deleted file mode 100644
index 40010aa242f61eda6c08a6daed3bb050824cc264..0000000000000000000000000000000000000000
diff --git a/android/src/main/res/values/attrs.xml b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/res/values/attrs.xml
deleted file mode 100644
index 54d250f341623b94bcb86ee02cb838a324b25070..0000000000000000000000000000000000000000
diff --git a/android/src/main/res/values/colors.xml b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/res/values/colors.xml
deleted file mode 100644
index f10bea37ece44f48741039d20041977d474b7b2f..0000000000000000000000000000000000000000
diff --git a/android/src/main/res/values-night/colors.xml b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/res/values-night/colors.xml
deleted file mode 100644
index 1d9e07ffd36becb1ec74df5498735edec35495d3..0000000000000000000000000000000000000000
diff --git a/android/src/main/res/xml/image_picker_provider_paths.xml b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/android/src/main/res/xml/image_picker_provider_paths.xml
deleted file mode 100644
index 975823ca11e9e9bf021d17f8662d9b7fe006c6b0..0000000000000000000000000000000000000000
diff --git a/app.plugin.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/app.plugin.js
deleted file mode 100644
index 6bacc141d48431ba636c2e25b16647061a4e9d11..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.d.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.d.ts
deleted file mode 100644
index 6c9e27f50043944f450e6c6485e153dca0dd4f96..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.d.ts.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.d.ts.map
deleted file mode 100644
index d4037a0279f2888f2425458db6164a734ce0776c..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.js
deleted file mode 100644
index b03e08a3c6afb7d4f0d85587559b67d7cc018205..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.js.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.js.map
deleted file mode 100644
index 12dab2ee01c7ab7759ea32028d81b4dc71c79558..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.web.d.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.web.d.ts
deleted file mode 100644
index e8fbb5fd1d48d8db25d20ad17b2528b681e0c409..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.web.d.ts.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.web.d.ts.map
deleted file mode 100644
index 737a56196d27428a9a253f57adce26b0395a79c4..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.web.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.web.js
deleted file mode 100644
index 94b31731782ab621405d8ce06a19af53addd19d0..0000000000000000000000000000000000000000
diff --git a/build/ExponentImagePicker.web.js.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ExponentImagePicker.web.js.map
deleted file mode 100644
index a1199961440661cebe6f88551814468209bd717c..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.d.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.d.ts
deleted file mode 100644
index c5522feeb876d74c9e81f5ee72edea3e038380fd..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.d.ts.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.d.ts.map
deleted file mode 100644
index 784cdddfc4ebb94196eec197f33210ba5277c786..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.js
deleted file mode 100644
index b3a42e628b4119ec56ffb0c88186f491bc60d0de..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.js.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.js.map
deleted file mode 100644
index 4a79f8a1ed7b08841cef13e221551e8162217446..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.types.d.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.types.d.ts
deleted file mode 100644
index 66aa4d4208ad513dce694a31167d8c01ac905d3a..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.types.d.ts.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.types.d.ts.map
deleted file mode 100644
index 51e272f0ddb5111218dc162f3dbd7c4d486b06c2..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.types.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.types.js
deleted file mode 100644
index 12859a8039ce98bd99c24ee9c1dc237b06d3ad05..0000000000000000000000000000000000000000
diff --git a/build/ImagePicker.types.js.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/ImagePicker.types.js.map
deleted file mode 100644
index 025ab1022ce53df1272688551099e34ed204bb2d..0000000000000000000000000000000000000000
diff --git a/build/utils.d.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/utils.d.ts
deleted file mode 100644
index d2c1836934fd7e33c5dd7911f7ab4c94b78701c2..0000000000000000000000000000000000000000
diff --git a/build/utils.d.ts.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/utils.d.ts.map
deleted file mode 100644
index 9011d9b5c30e508eba3586ea9bf284757c97d690..0000000000000000000000000000000000000000
diff --git a/build/utils.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/utils.js
deleted file mode 100644
index 31a0ac6c34e0198be723593cea3e6e35e92e9258..0000000000000000000000000000000000000000
diff --git a/build/utils.js.map b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/build/utils.js.map
deleted file mode 100644
index 190c63f0938de1cc04a6143384fd05eeccda20aa..0000000000000000000000000000000000000000
diff --git a/expo-module.config.json b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/expo-module.config.json
deleted file mode 100644
index c947cbe7a68fff0c93d440076e9d777c1b0e46c0..0000000000000000000000000000000000000000
diff --git a/ios/ExpoImagePicker.podspec b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ExpoImagePicker.podspec
deleted file mode 100644
index 2b145acdafa1e91ca1e3f5a22c975b27f57d8621..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerExceptions.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerExceptions.swift
deleted file mode 100644
index a0810e5ff9739ac43062b8719bf55db98f52c90a..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerHandler.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerHandler.swift
deleted file mode 100644
index 476910ded2a9f53ce18563b3f8c005717cb451e4..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerModule.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerModule.swift
deleted file mode 100644
index 4b7bf34fb79ed99a6992b2df983488746b714856..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerOptions.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerOptions.swift
deleted file mode 100644
index bdaf98fdb986672101d2e9f83dc1b09aeb0b6e03..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerPermissionRequesters.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerPermissionRequesters.swift
deleted file mode 100644
index 41ec2e9abe8ffe3084c887411fd0ccfdbe4933b2..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerResponse.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerResponse.swift
deleted file mode 100644
index 21b9bb116d585d7fffc9e4aa0c768cb9e8c11036..0000000000000000000000000000000000000000
diff --git a/ios/ImagePickerUtils.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImagePickerUtils.swift
deleted file mode 100644
index cfd0bcbb37918d5f634836007c25958774e48a82..0000000000000000000000000000000000000000
diff --git a/ios/ImageUtils.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/ImageUtils.swift
deleted file mode 100644
index d9d1cd4d8402a7601d84fa593ccc1c0f43021b59..0000000000000000000000000000000000000000
diff --git a/ios/MediaHandler.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/MediaHandler.swift
deleted file mode 100644
index a0e3a442d55e63dcbad68dfd37dffc8a2d942572..0000000000000000000000000000000000000000
diff --git a/ios/NSItemProvider+Async.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/NSItemProvider+Async.swift
deleted file mode 100644
index 08cdf4ca9d97669112ce3ac2bb352630a3c04668..0000000000000000000000000000000000000000
diff --git a/ios/StatusBarVisibilityController.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/StatusBarVisibilityController.swift
deleted file mode 100644
index 907af5884fa27f44c1ae039497fd24b590693fa3..0000000000000000000000000000000000000000
diff --git a/ios/UIImage+fixOrientation.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/UIImage+fixOrientation.swift
deleted file mode 100644
index 22849c2d46347b42e2a0bcea72bb0d1f631c574d..0000000000000000000000000000000000000000
diff --git a/ios/UIImagePickerControllerExtentsion.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/UIImagePickerControllerExtentsion.swift
deleted file mode 100644
index 865a59db22b8baefc09373a81629d8dfe08bf402..0000000000000000000000000000000000000000
diff --git a/ios/VideoUtils.swift b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/ios/VideoUtils.swift
deleted file mode 100644
index f78c50dfe58e3e352029bb2c9c4224611c92e6b6..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar
deleted file mode 100644
index 5d9421ad834e4c95995bc86096c6837217672f56..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.md5 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.md5
deleted file mode 100644
index 397f796dbfb9bcef93f690c07c538005d5017166..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.sha1 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.sha1
deleted file mode 100644
index 75348bafdff5ef98a5cce0e498eb0b2af7a055e3..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.sha256 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.sha256
deleted file mode 100644
index 99b20cff72a78110653faa4a712c4607511456b6..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.sha512 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12-sources.jar.sha512
deleted file mode 100644
index 27147ff1680e8a2f9488b13a3d8c822ff5826838..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar
deleted file mode 100644
index d84e0b2eb8483088a455f3c03199cea8164655fe..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.md5 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.md5
deleted file mode 100644
index c8360c03847a80c81f3d1e6e562a754c25c9c618..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.sha1 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.sha1
deleted file mode 100644
index 6a955a2f17b296737e3dbbd61601b7c12296d8b9..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.sha256 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.sha256
deleted file mode 100644
index 36c654b276c491d78ff4111b471ca3cea0a75812..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.sha512 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.aar.sha512
deleted file mode 100644
index 99cf5b815ffaf6e21e1c06ea8426c3431afb3b99..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module
deleted file mode 100644
index f8975ad703766bc882ebe57fcf062829f1a2df4d..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.md5 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.md5
deleted file mode 100644
index d1cbd94f5461fe442e1fde91b8a58a77994f0d67..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.sha1 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.sha1
deleted file mode 100644
index 4d119c41fd679f6b00f058ac390e050d7483f8bc..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.sha256 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.sha256
deleted file mode 100644
index 006c2b734241cb7da8e1c7c835396ca6053541eb..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.sha512 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.module.sha512
deleted file mode 100644
index b94f2b7a7b129309c3ec75aa55f3fb4ca9c16f0d..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom
deleted file mode 100644
index 0adf16104f29380edc12068abeed66d86e8176a4..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.md5 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.md5
deleted file mode 100644
index 0b860eede073e3218a4eb32f33f1fd5841c54d75..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.sha1 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.sha1
deleted file mode 100644
index 140f1cdae82ff5a69e399b08b30bfdfe7104d8ed..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.sha256 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.sha256
deleted file mode 100644
index 731cb5c0b740aadd82f0234abc28031fb84790ee..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.sha512 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/55.0.12/expo.modules.imagepicker-55.0.12.pom.sha512
deleted file mode 100644
index 55a0b52c895aab2a8e73fae3910afdfc2bce9012..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml
deleted file mode 100644
index f06ff7c3291ecd6e37f6e711e1e304b05eda2d71..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.md5 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.md5
deleted file mode 100644
index 6d3b83e1bf178585176e1abcb6c25e34a9dd9103..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.sha1 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.sha1
deleted file mode 100644
index 5a0da121861efbd99fd07bc2dd3580bc9e3b2d99..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.sha256 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.sha256
deleted file mode 100644
index f23b9b041204f41097db8238e8a4b3777cef5d21..0000000000000000000000000000000000000000
diff --git a/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.sha512 b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/local-maven-repo/host/exp/exponent/expo.modules.imagepicker/maven-metadata.xml.sha512
deleted file mode 100644
index 6b84a261bdae55e04d32aecea51a961018a05cd2..0000000000000000000000000000000000000000
diff --git a/package.json b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/package.json
deleted file mode 100644
index 442242a0a93ab7f704d77554736512829305ec33..0000000000000000000000000000000000000000
diff --git a/plugin/build/withImagePicker.d.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/plugin/build/withImagePicker.d.ts
deleted file mode 100644
index 659dd225737c39c7e67d5fb00a5908399aedb8d3..0000000000000000000000000000000000000000
diff --git a/plugin/build/withImagePicker.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/plugin/build/withImagePicker.js
deleted file mode 100644
index 7fbe663360b69120ae816b156f9faf996ac36d0d..0000000000000000000000000000000000000000
diff --git a/plugin/jest.config.js b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/plugin/jest.config.js
deleted file mode 100644
index e539b3b187ee4fe173e97c78cc93df51010366b9..0000000000000000000000000000000000000000
diff --git a/plugin/src/withImagePicker.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/plugin/src/withImagePicker.ts
deleted file mode 100644
index ea38b3c6fa9d9e12e9d92c64cb438390707d3b70..0000000000000000000000000000000000000000
diff --git a/plugin/tsconfig.json b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/plugin/tsconfig.json
deleted file mode 100644
index 354bddb4336eb1af6183035c78f63d690bfcacba..0000000000000000000000000000000000000000
diff --git a/src/ExponentImagePicker.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/src/ExponentImagePicker.ts
deleted file mode 100644
index 8f7d36b70fa322914d509b308824f84fc4cb7b8f..0000000000000000000000000000000000000000
diff --git a/src/ExponentImagePicker.web.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/src/ExponentImagePicker.web.ts
deleted file mode 100644
index 6c910789e724db65620a0852575abac914c6fecd..0000000000000000000000000000000000000000
diff --git a/src/ImagePicker.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/src/ImagePicker.ts
deleted file mode 100644
index 656b6466c50ea43ce19f0b784f4ce946242304a3..0000000000000000000000000000000000000000
diff --git a/src/ImagePicker.types.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/src/ImagePicker.types.ts
deleted file mode 100644
index 1440510d6d3470a4c8ad885ebd7cc68356d60d3b..0000000000000000000000000000000000000000
diff --git a/src/utils.ts b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/src/utils.ts
deleted file mode 100644
index 85f6d6969f4e55b8cec0994cc7b46272e191b3ae..0000000000000000000000000000000000000000
diff --git a/tsconfig.json b/Users/louisciccone/.bun/install/cache/expo-image-picker@55.0.12@@@1/tsconfig.json
deleted file mode 100644
index cbe9e1971c70b84cad0dab6ba1e5b93b599f6fb5..0000000000000000000000000000000000000000
